### PR TITLE
Update media facade : add orginal and replace thumbnails with alterna…

### PR DESCRIPTION
**Pull request by ngilain**
…tives

### CODE REVIEW

**Comment by nicolasThal** [comment]
:+1:

File: MediaAdmin/FileAlternatives/FileAlternativesStrategyInterface.php [hunk 1]
```diff
@@ -35,6 +35,15 @@ public function generateThumbnail(MediaInterface $media);
     public function generateAlternatives(MediaInterface $media);
 
     /**
+     * Get alternatives from $media
+     * 
+     * @param $media
```
Comment (alavieille) [inline MediaAdmin/FileAlternatives/FileAlternativesStrategyInterface.php:40]: param MediaInterface $media

File: MediaAdmin/FileAlternatives/Strategy/AbstractFileAlternativesStrategy.php [hunk 2]
```diff
@@ -29,6 +29,18 @@ public function generateAlternatives(MediaInterface $media)
     }
 
     /**
+     * Get alternatives from $media
+     * 
+     * @param $media
+     * 
+     * @return array
+     */
+    public function getAlternatives(MediaInterface $media)
```
Comment (alavieille) [inline MediaAdmin/FileAlternatives/Strategy/AbstractFileAlternativesStrategy.php:38]: param MediaInterface $media

File: MediaAdmin/FileAlternatives/Strategy/ImageStrategy.php [hunk 3]
```diff
@@ -108,6 +108,24 @@ protected function generateAlternative($fileName, $formatName, array $formatSize
     }
 
     /**
+     * Get alternatives from $media
+     * 
+     * @param $media
```
Comment (alavieille) [inline MediaAdmin/FileAlternatives/Strategy/ImageStrategy.php:113]: param MediaInterface $media

**Pull request merged**